commands/issue/update/issue_update.go

```diff
@@ -122,6 +187,7 @@ func NewCmdUpdate(f *cmdutils.Factory) *cobra.Command {
 	issueUpdateCmd.Flags().BoolP("public", "p", false, "Make issue public")
 	issueUpdateCmd.Flags().BoolP("confidential", "c", false, "Make issue confidential")
 	issueUpdateCmd.Flags().StringP("milestone", "m", "", "title of the milestone to assign, pass \"\" or 0 to unassign")
+	issueUpdateCmd.Flags().StringArrayP("assignee", "a", []string{}, "assign users via username, prefix with '!' or '-' to remove from existing assignees, '+' to add, otherwise replace existing assignees with given users")
```

Comment:
We should use `StringSliceP` instead as it allows both comma-separated and multiple usages
https://pkg.go.dev/github.com/spf13/pflag@v1.0.5#StringArray
https://pkg.go.dev/github.com/spf13/pflag@v1.0.5#StringSlice

With stringslice, this is allowed: `--assignee "user1,user2" --assignee "user3"`